### PR TITLE
Issue 1929: Oauth2-proxy v7.4.0 is not using alpine:3.16 as it is wri…

### PR DESCRIPTION
**Pull request by miguelborges99**
## Description

Upgrade alpine and dependencies

## Motivation and Context

Provide the latest CVE fixes and solve issue #1929 

## How Has This Been Tested?

Under test

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [X] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

**Review by JoelSpeed — DISMISSED** [review]
Thanks

**Comment by JoelSpeed** [comment]
Could you please resolve the conflicts and add a changelog and then we can merge this

**Comment by miguelborges99** [comment]
> Could you please resolve the conflicts and add a changelog and then we can merge this

Conflits were solved. Sorry, for the time it took, I was rather busy.

**Review by JoelSpeed — APPROVED** [review]
Looks great, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -907,6 +907,7 @@ N/A
     - `-bitbucket-team` Restrict authorization to users that are part of this Bitbucket team
 - [#211](https://github.com/oauth2-proxy/oauth2-proxy/pull/211) Switch from dep to go modules (@steakunderscore)
 - [#145](https://github.com/oauth2-proxy/oauth2-proxy/pull/145) Add support for OIDC UserInfo endpoint email verification (@rtluckie)
+- [#2013](https://github.com/oauth2-proxy/oauth2-proxy/pull/2013) Upgrade alpine to version 3.17.2 and library dependencies (@miguelborges99)
```
Comment (JoelSpeed) [inline CHANGELOG.md:910]: Oops, this was in the wrong place, will need to move that up to the top

**Pull request merged**